src/pages/Lending/Pool/Pool.tsx

```diff
@@ -56,31 +69,112 @@ const PoolHeader = () => {
 const flexDirPool: ResponsiveValue<Property.FlexDirection> = { base: 'column', lg: 'row' }
 
 export const Pool = () => {
-  const { pid } = useParams<{ pid?: string }>()
+  const poolAssetId = useRouteAssetId()
+  const asset = useAppSelector(state => selectAssetById(state, poolAssetId))
+
   const translate = useTranslate()
   const [value, setValue] = useState<number | string>()
 
+  const sellAssetMarketData = useAppSelector(state => selectMarketDataById(state, poolAssetId))
+  const accountId =
+    useAppSelector(state =>
+      selectFirstAccountIdByChainId(state, fromAssetId(poolAssetId).chainId),
+    ) ?? ''
+
+  const lendingPositionQueryKey: [string, { accountId: AccountId; assetId: AssetId }] = useMemo(
+    () => ['thorchainLendingPosition', { accountId, assetId: poolAssetId }],
+    [accountId, poolAssetId],
+  )
+  const repaymentLockQueryKey = useMemo(() => ['thorchainLendingRepaymentLock'], [])
+
+  const { data: lendingPositionData, isLoading: isLendingPositionDataLoading } = useQuery({
+    queryKey: lendingPositionQueryKey,
+    queryFn: async ({ queryKey }) => {
+      const [, { accountId, assetId }] = queryKey
+      const position = await getThorchainLendingPosition({ accountId, assetId })
+      return position
+    },
+    select: data => {
+      // returns actual derived data, or zero's out fields in case there is no active position
+      const collateralBalanceCryptoPrecision = fromThorBaseUnit(data?.collateral_current).toString()
+
+      const collateralBalanceFiatUserCurrency = fromThorBaseUnit(data?.collateral_current)
```

Comment:
derp, this should *always* be denominated in USD

```suggestion
      const collateralBalanceUsd = fromThorBaseUnit(data?.collateral_current)
```